tvaultuiv2/src/views/private/safe/components/Permissions/components/User/index.js

```diff
@@ -114,70 +116,123 @@ const User = (props) => {
     onNoDataAddClicked,
     onCancelClicked,
   } = props;
-
+  const [responseType, setResponseType] = useState(null);
+  const [toastMessage, setToastMessage] = useState('');
+  const [editUser, setEditUser] = useState('');
+  const [editAccess, setEditAccess] = useState('');
+  const [editPermission, setEditPermission] = useState(false);
   const isMobileScreen = useMediaQuery(small);
-  const onSubmit = (search, radio) => {
+
+  const onSubmit = (user, access) => {
     const value = {
-      name: search,
-      last_updated: '2days ago',
-      permission: radio,
+      access,
+      path: 'shared/test-safe3',
+      username: user.toLowerCase(),
     };
     onSaveClicked(value);
   };
+
+  const onDeleteClick = (username) => {
+    const payload = {
+      path: 'shared/test-safe3',
+      username,
+    };
+    apiService
+      .deleteApiCall('/vault/v2/ss/sdb/user', payload)
```

Comment:
API path should be part of apiServices.js

Reply:
fixed